ChatBot/ChatBot/Controller/ChatRoomViewController.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  ChatRoomViewController.swift
+//  ChatBot
+//
+//  Created by Tacocat on 1/1/24.
+//
+
+import UIKit
+
+final class ChatRoomViewController : UIViewController {
+    private typealias DataSource = UICollectionViewDiffableDataSource<Section, ChatBubble>
+    
+    enum Section {
+        case main
+    }
+    
+    private var chats: [ChatBubble] = []
+    private let chatService = ChatService()
+    private var dataSource: DataSource?
+    
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        return stackView
+    }()
```

Comment:
self 를 참조하지 않는 뷰라면 lazy 를 사용할 필요가 없을 것 같아요.

```suggestion
    private let mainStackView: UIStackView = {
        let stackView = UIStackView()
        stackView.axis = .vertical
        stackView.alignment = .fill
        return stackView
    }()
```

Reply:
말씀처럼 self를 참조하지 않는 상황이고 항상 사용하는 뷰라서 lazy하게 선언할 필요가 없는 것 같습니다. 수정하였습니다.

Reply:
[661ab6f](https://github.com/tasty-code/ios-chat-bot/pull/23/commits/661ab6fa6c69211209be963e1223ad917d64a8f1) 입니다 :)